xrdsst/controllers/client.py

```diff
@@ -206,7 +239,19 @@ def remote_import_tls_certificate(self, ss_api_config, tls_certs, client_conf):
                 for tls_cert in tls_certs:
                     self.remote_add_client_tls_certificate(tls_cert, clients_api, client)
         except ApiException as find_err:
-            BaseController.log_api_error(ClientController.CLIENTS_API_FIND_CLIENTS, find_err)
+            BaseController.log_api_error("ClientsApi->find_client", find_err)
+
+    def remote_unregister_client(self, ss_api_config, security_server, clientsId):
```

Comment:
1. remove self from parameters as it is not used and add "@staticmethod" tag to this method

2.  change "clientId" to "client_id" as Python requires using snake-case instead of camel-case, which is used in Java